files/docker/node/dockerfile_bin

```diff
@@ -19,10 +19,11 @@ ENV \
     CNODE_HOME=/opt/cardano/cnode \
     CARDANO_NODE_SOCKET_PATH=$CNODE_HOME/sockets/node.socket \
     PATH=/opt/cardano/cnode/scripts:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/home/guild/.local/bin \
-    GIT_SSL_CAINFO=/etc/ssl/certs/ca-certificates.crt
+    GIT_SSL_CAINFO=/etc/ssl/certs/ca-certificates.crt \
+    G_BRANCH=${GUILD_DEPLOY_BRANCH}
 
 RUN apt-get update && apt-get install --no-install-recommends -y locales apt-utils sudo \
-    && apt install -y curl wget gnupg git udev \
+    && apt install -y curl wget gnupg git udev jq \
```

Comment:
Why install `jq` on line 26 without removing the installation from line 72?

Reply:
this is a valid concern. agree'd that is should be removed from line 73.

Reply:
ref: https://github.com/Fuma419/guild-operators/commit/1cd58dd1989960d0574680a6bf978e86db2a64af

Reply:
:+1: 

I think I found the changes causing the breakage you experienced. It appears a month ago in commit 00807de `jq` was added to `guild-deploy.sh` to parse variables from `node-deps.json` and indeed before line 73 `guild-deploy.sh` is used before `jq` gets installed. 

For tracking purposes I suggest when a problem is found to create a unique issue in the repository for it. When opening the PR add `closes XXX` for each issue to the description. While the PR itself would not follow separations of concern, it at least allows the reviewer to discern the reason for changes that appear unrelated to exist in the same PR without inquiring about each.